src/pages/saksbehandling/sakintro/AvsluttBehandling.tsx

```diff
@@ -49,9 +49,25 @@ const AvsluttBehandling = (props: { sak: Sak }) => {
         validateOnChange: hasSubmitted,
     });
 
+    const lukketSøknadBrev = useCallback(() => {
+        if (RemoteData.isPending(lukketSøknadBrevutkastStatus) || !formik.values.lukkSøknadType) {
+            return;
+        }
+        dispatch(
+            hentLukketSøknadBrevutkast({
+                søknadId: urlParams.soknadId,
+                lukketSøknadType: formik.values.lukkSøknadType,
+            })
+        ).then((action) => {
+            if (hentLukketSøknadBrevutkast.fulfilled.match(action)) {
+                window.open(action.payload.objectUrl);
+            }
+        });
+    }, [formik.values.lukkSøknadType]);
+
     const søknad = props.sak.søknader.find((s) => s.id === urlParams.soknadId);
 
-    if (RemoteData.isSuccess(søknadsbehandlingAvsluttet) || (søknad && søknad.lukket !== null)) {
+    if (RemoteData.isSuccess(søknadsbehandlingAvsluttetStatus) || (søknad && søknad.lukket !== null)) {
```

Comment:
Kan förenkla RHS pitteelittttt 😄 
```suggestion
    if (RemoteData.isSuccess(søknadsbehandlingAvsluttetStatus) || (søknad?.lukket !== null)) {
```

Reply:
👍